MACM/MACMInitialization.py

```diff
@@ -40,6 +40,7 @@
 import datetime as dt
 import pickle
 import time
+import os
 
 
 ACTIVITY_THRESHOLD = {'twitter': 17, 'youtube': 2}
```

Comment:
@deamonpog activity threshold `dict` should be made a parameter as well, please.